Source/Fuse.Share/ios/iOSShareImpl.uno

```diff
@@ -2,24 +2,63 @@ using Uno.Compiler.ExportTargetInterop;
 using Uno.Threading;
 namespace Fuse.Share
 {
-	public extern(iOS) class iOSShareImpl
+	internal extern(iOS) class iOSShareImpl
 	{
-		[Foreign(Language.ObjC)]
 		public static void ShareText(string text, string description)
+		{
+			if (IsIPad())
+				Fuse.Diagnostics.UserWarning("iPad requires a position as the spawn origion of the Share popover", text + " - " + description);
```

Comment:
There is no reasonable default position. iOS design guidelines requires the popover to have a sensible origin. On iPad the popover looks like this: ![ipad](http://ivomynttinen.com/content/3-blog/20150928-ios-design-guidelines/ios-popover.jpg)